packages/api-client/src/components/ApiClient/Request/Request.vue

```diff
@@ -251,10 +251,11 @@ const { activeRequest, readOnly } = useRequestStore()
   background: transparent;
   outline: none;
   border: none;
+  -moz-appearance: none;
```

Comment:
I think we might have autoprefixer doing this for us 🤔

Reply:
i couldnt see it autoprefixer doing this for us in older browsers.

Reply:
maybe it was the with the software i was testing tho, but let's get this in to fix and i'll make a ticket to audit autoprefixer on older browsers for `appearance: none`